### ISSUE TITLE
Not all resources are shown in popup

### ISSUE THREAD

**Issue by GoogleCodeExporter** [bliu886/scriptno#51]
```
What steps will reproduce the problem?

1. Go to www.sport5.co.il
2. Only www.sport5.co.il is listed as a blocked resource in the popup

What is the expected output? What do you see instead?

There are other resources being blocked, such as "homepage.sport5.co.il", as 
can be seen in the network tab of the developer tools (Ctrl+Shift+I).

What version of the product are you using? On what operating system?

ScriptNo 1.0.5.52.
Latest Chromium (Build 118091 at the time of writing).

Please provide any additional information below.

If the option Respect Same-Domain is enabled, both "www.sport5.co.il" and 
"homepage.sport5.co.il" are not being blocked, as expected.
In addition, both are shown in the popup, as expected.
```

Original issue reported on code.google.com by `adam...@gmail.com` on 18 Jan 2012 at 8:36

**Comment by GoogleCodeExporter** [comment]
```
I'm experiencing this issue on a lot of websites during everyday use. 
Basically, this makes it impossible to use many of those sites, the only 
solution is disabling ScriptNo altogether.

Running Experimental 1.05.52 under Chrome 18.0.1010.1 
```

Original comment by `furex.fu...@gmail.com` on 21 Jan 2012 at 4:42

**Comment by GoogleCodeExporter** [comment]
```
Same here.
At current state, extensions is pretty much unusable, so I had to disable it as 
well.
Not that I was using the non-experimental version (due to issues with AdBlock 
Plus).
```

Original comment by `adam...@gmail.com` on 22 Jan 2012 at 8:07

**Comment by GoogleCodeExporter** [comment]
```
* Note that I was using... *
```

Original comment by `adam...@gmail.com` on 22 Jan 2012 at 10:17

**Comment by GoogleCodeExporter** [comment]
Original comment by `andr...@gmail.com` on 9 Feb 2012 at 4:20

**Comment by GoogleCodeExporter** [comment]
```
The 3rd party blocked domains not showing in popup in ScriptNo 1.0.5.52 has 
been fixed in 1.0.6.0 (and later). I had the same issue with Chromium 
dev-channel before updating today.
```

Original comment by `sah...@gmail.com` on 18 Feb 2012 at 11:31

**Comment by GoogleCodeExporter** [comment]
Original comment by `andr...@gmail.com` on 19 Feb 2012 at 7:31
- Changed state: **Done**
